Tests/MongoSwiftTests/MongoClientTests.swift

```diff
@@ -150,3 +166,15 @@ final class MongoClientTests: MongoSwiftTestCase {
         try closeFuture.wait()
     }
 }
+
+class TestEventLoop: Equatable {
```

Comment:
This wrapper is just a helper for comparing event loops in the tests since `EventLoop` is a protocol. Is there a another way to be able to check the event loop used in the operation is the same as the one specified in `bound()`? Or is this fine?

Reply:
Nimble supplies matchers for using the `===` operator directly, so you can just use those in the test.

https://github.com/Quick/Nimble#identity

Reply:
thanks!